frontend/cypress/tests/chartView/chartView_percentage.cy.js

```diff
@@ -67,6 +67,8 @@ describe('sort by contribution', () => {
     cy.get('#summary .summary-chart__title--percentile')
       .should('exist');
 
+    cy.get('#summary-wrapper').scrollTo('top');
+
```

Comment:
This was a strange bug where mui-select.grouping is not visible specifically for this test case and not for the previous test case. I added this to scroll to the top of the summary wrapper so it is visible.